bitar/Cargo.toml

```
@@ -26,11 +26,16 @@ bytes = "1.1"
 rust-lzma = { version = "0.5", optional = true }
 zstd = { version = "0.9", optional = true }
 async-trait = "0.1"
+tempfile = { version="3.2.0", optional = true }
+num_cpus = { version="1.13.1", optional = true }
```

```suggestion
tempfile = { version="3.2.0", optional = true }
num_cpus = { version="1.13.1", optional = true }
```
```suggestion
tempfile = { version="3.2", optional = true }
num_cpus = { version="1.13", optional = true }
```
I think libraries typically specify dependencies with minor version, at least as long as the minimum patch version doesn't matter. Don't think it matters much though.

---

Removed the minor version from the dependencies